common/CMakeLists.txt

```diff
@@ -180,6 +180,7 @@ target_include_directories(${LIB_NAME} PUBLIC
 )
 
 target_link_libraries(${LIB_NAME} Boost::boost)
+target_link_libraries(${LIB_NAME} Eigen3::Eigen)
```

Comment:
Eigen3::Eigen can be appended to the line above, after Boost::boost.

Reply:
Done with [7c692f0](https://github.com/PointCloudLibrary/pcl/pull/5613/commits/7c692f05fa374ec5099170253b4d16b0124b73a7)